PharmacyManagement/HumanManage/NewAccount.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Data;
+using System.Data.SqlClient;
+using System.Windows.Forms;
+using DevExpress.XtraEditors;
+using PharmacyManagement.DB_query;
+
+namespace PharmacyManagement.HumanManage
+{
+    public partial class NewAccount : XtraForm
+    {
+        PharmacyMgtDatabase dataTable = new PharmacyMgtDatabase();
+        public NewAccount()
+        {
+            dataTable.OpenConnection();
+            InitializeComponent();
+            cboUserRole.SelectedIndex = 0;
+        }
+
+        private void NewAccount_Load(object sender, EventArgs e)
+        {
+            txtAccountID.Enabled = false;
+            ClearAllField();
+        }
+
+        private void btnAdd_Click(object sender, EventArgs e)
+        {
+            if (ValidateInput())
+            {
+                string sql = @"INSERT INTO ACCOUNT (Username, UserPassword, UserRole) 
```

Comment:
why dont we remove ID textbox? Either way, we dont use it

Reply:
done